src/ui/Views/Organisation/Show.cshtml

```diff
@@ -73,7 +73,7 @@
         {
           <tr class="govuk-table__row">
             <td class="govuk-table__cell">
-              <a href="@Url.Action("show", "course", new {instCode = Model.InstitutionId, accreditingProviderId = course.AccreditingProviderId, ucasCode = course.CourseCode})" class="govuk-link">@course.Name (@course.CourseCode)</a>
+              <a href="@Url.Action("show", "course", new {instCode = Model.InstCode, accreditingInstCode = course.AccreditingProviderId, courseCode = course.CourseCode})" class="govuk-link">@course.Name (@course.CourseCode)</a>
```

Comment:
can i reject `accreditingInstCode = course.AccreditingProviderId`

Reply:
that's coming from API as well :)